src/pages/MainPage/MainPage.tsx

```diff
@@ -157,43 +169,62 @@ export const MainPage: React.FC = (): React.ReactElement => {
   }, []);
 
   return (
-    <div className='font-ibm max-w-[375px] min-w-[375px]'>
-      <Header />
+    <div className={`font-ibm ${isDesktop ? 'max-w-full px-16' : 'max-w-[375px] min-w-[375px]'}`}>
```

Comment:
I think we can use css breakpoints here🤔
https://tailwindcss.com/docs/responsive-design